Bench.php

```diff
@@ -311,6 +315,43 @@ public function getLongest()
     }
 
     /**
+     * Pause all marks.
+     *
+     * @return  array
+     */
+    public function pause()
+    {
+        $runningMarks = [];
+
+        foreach ($this as $mark) {
+            if (true === $mark->isRunning() && false === $mark->isPause()) {
+                $runningMarks[] = $mark;
+            }
+        }
+
+        // short loop = reduce delta between pauses
+        // same loop as in resume()
```

Comment:
Remove the comment please :smiley: We clearly understand this part of the code :-].